watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1920,8 +1919,19 @@ def calculate_scaling_factors(self):
                     )
                     iscale.set_scaling_factor(v, sf)
 
+        for (t, x, p, j), v in self.flux_mol_phase_comp.items():
+            if iscale.get_scaling_factor(v) is None:
                 if comp.is_solute():
                     # Todo: revisit later
+                    physical_rejection_factor = (
+                        self.permeate_side[t, x].radius_stokes_comp[j].value ** 2
+                        / self.radius_pore.value**2
+                    )
+                    # will be 9, 99,999 etc for 1,2,3 valance
+                    rejection_factor = (
+                        10 ** abs(self.permeate_side[t, x].charge_comp[j].value) - 1
+                    ) / 10 ** abs(self.permeate_side[t, x].charge_comp[j].value)
```

Comment:
I am trying to bring these into my PR and see what the result will be. However, is rejection_factor making much difference? The comment says will be 9, 99, 999 for 1,2,3 valence, but as written, it would be 0.9, 0.99, 0.999, leading me to believe that `rejection_factor might not be critical. Sanity check?

Reply:
Ah- it should actually make a difference since you are doing `(1-rejection_factor)` below.

Reply:
Yea. That was not the most clear note. should of added the %. We also need to think about how to deal with "physical hinderence".....wait. dont we have an expression for that in NF model already ....

Reply:
`lambda_comp ` is the ratio between stokes radius of solute and membrane pore radius, so we technically could've used that in place of your manual writing of it although it is an expression with a smooth_min function in it, so I didn't press for it. We do have convective and diffusive hindrance factors in the NF model if that's what you were hinting at.

Reply:
Yeah, I actually tested out using diffus_pore_comp, but it made a minor change to unscaled tolerances. So I did not update it. 

The issue still remains for me, that if I add low concentration of ion like Sr or Ba, my dual infeasibility is at about 7e-6, ideally, we want to push that to 1e-8, like in the current nf flowsheet. but there is still something missing in the puzzle.  I wounder if we are focusing on the wrong scaling. One other thought came to mind, that we don't properly scale concentrations at membrane water interface, which would be higher than bulk...maybe that is worth looking at next for me. 

I will keep this PR up so we can use it as a test bed, there must be a better way to do this scailing.

Reply:
I will put this PR into draft mode to avoid confusion by other potential reviewers.

Reply:
Great. Go ahead and push PR#1136 first. I want to keep this up so I can test linux/windows impacts, and propose new scailing approaches if there is somethhing I figure out that's cleaner/better.